set flatten of properties to false

matthiasschaub opened the pull request:
### Description
Per default properties of the GeoJSON response are not flattend. Default of API request parameter `flatten` is `False`.

### Corresponding issue
None

### New or changed dependencies
- none

### Checklist
- [ ] I have updated my branch to `main` (e.g. through `git rebase main`)
- [ ] My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing
- [ ] I have commented my code
- [ ] I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)
- [ ] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CHANGELOG.md)

joker234 reviewed (commented):
Look good. Maybe look over the examples and documentation once and search for the word `flatten`. Some examples could be set to the default value.

joker234 reviewed (commented):
`request_models.py` has some `flatten=True` examples

matthiasschaub commented:
> Look good. Maybe look over the examples and documentation once and search for the word `flatten`. Some examples could be set to the default value.

Thanks for the review. I changed the value of the flatten parameter to represent the default value (Flase) in docs and API request examples.
Also I removed the unflatten function defintion and usage in the JavaScript files of the website.
